caddyhttp/proxy/reverseproxy.go

```diff
@@ -252,8 +252,25 @@ func (rp *ReverseProxy) copyResponse(dst io.Writer, src io.Reader) {
 	io.CopyBuffer(dst, src, buf.([]byte))
 }
 
+var skipHeaders = map[string]struct{}{
+	"Content-Type":        {},
+	"Content-Disposition": {},
+	"Accept-Ranges":       {},
+	"Set-Cookie":          {},
+	"Cache-Control":       {},
+	"Expires":             {},
+}
+
 func copyHeader(dst, src http.Header) {
 	for k, vv := range src {
+		if _, ok := dst[k]; ok {
+			// some predefined headers (e.g. MIME plugin) shoule skip.
```

Comment:
I'm not 100% sure this has anything to do with the mime plugin, maybe we should use this comment to refer to issue #1086 instead.